src/lsrs/web/converter/converters.ts

```diff
@@ -0,0 +1,69 @@
+import {AcceptorInfo} from "../../core/model/file-browser-api";
+import {FileInput, MIMEType} from "../../core/model/file-input";
+import {UploadedFileCreateAttributes, UploadedFileUpdateAttributes} from "../../core/model/uploaded-file";
+import {DirectoryModel, FileModel, FileUploadRequestModel, UpdateFileMetadataRequestModel} from "../model/files";
+
+/**
+ * Converts an UploadedFileCreateAttributes object to FileModel.
+ *
+ * @param uploadedFile source UploadedFileCreateAttributes object
+ * @returns converted object as FileModel
+ */
+export function convertUploadedFile(uploadedFile: UploadedFileCreateAttributes): FileModel {
+
+    return {
+        originalFilename: uploadedFile.originalFilename,
+        acceptedAs: uploadedFile.acceptedAs,
+        description: uploadedFile.description,
+        path: uploadedFile.path,
+        reference: `/${uploadedFile.pathUUID}/${uploadedFile.storedFilename}`
+    }
+}
+
+/**
+ * Converts an AcceptorInfo object to DirectoryModel.
+ *
+ * @param acceptor source AcceptorInfo object
+ * @returns converted object as DirectoryModel
+ */
+export function convertAcceptor(acceptor: AcceptorInfo): DirectoryModel {
+
+    return {
```

Comment:
Would it make sense to simply cast the acceptor and return that? Since the properties seem to match up.

Reply:
Not exactly, but maybe I could that.. Will check!

Reply:
Yep, it did make sense, dropped the converter and modified the AcceptorInfo internal domain class. Thanks for the suggestion!